src/utils/index.ts

```diff
@@ -8,6 +8,8 @@ import {
 import { getISOTime, getMeetingDays } from './time';
 const isAdmin = (user: FullUser) => user.role === Role.ADMIN;
 
+const defaultPref = 2;
```

Comment:
Is this just in case if the default_pref value doesn't load from env?

Reply:
Yup